metadata.json

```diff
@@ -17,5 +17,14 @@
       "name": "puppetlabs/apt",
       "version_requirement": ">= 1.4.0"
     }
+  ],
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+        "12.04",
+        "14.04"
```

Comment:
We should probably start testing this on 16.04, because future is now and stuff.